types/incidentcreationconditions.pp

```diff
@@ -0,0 +1,26 @@
+# The possible incident creation conditions. Here's what each condition means:
+#   'always'
+#       Always create an incident for _all_ reports
+#   'never'
```

Comment:
Also went ahead and added `never` since it made sense now as an incident creation condition. The edge-case I talked about in previous PR (`['always', 'never']`) isn't as weird anymore b/c it's easy to reason about under the incident creation condition typename (we'll always have an incident created b/c the `always` condition is satisfied for all reports).